datahub-web-react/src/app/entity/group/GroupHeader.tsx

```diff
@@ -0,0 +1,46 @@
+import styled from 'styled-components';
+import React from 'react';
+import { Space, Typography } from 'antd';
+import CustomAvatar from '../../shared/avatar/CustomAvatar';
+
+type Props = {
+    name?: string | null;
+    email?: string | null;
+    description?: string | null;
+};
+
+const Row = styled.div`
+    display: inline-flex;
+`;
+
+const AvatarWrapper = styled.div`
+    margin-right: 32px;
+`;
+
+export default function GroupHeader({ name, description, email }: Props) {
+    return (
+        <>
+            <Row>
+                <AvatarWrapper>
+                    <CustomAvatar size={100} photoUrl={undefined} name={name || undefined} />
```

Comment:
is the photoUrl always undefined? If so, why include it?

Reply:
You're right - we shouldn't need this !

Reply:
On second thought. I'm leaving this because we should fill it in the model, just not in this PR. Added a TODO in the code.